### PR TITLE
forcing task to run in check mode

### PR DESCRIPTION
**Pull request by krknopp**
Adding this line allows the roles to finish in check mode.

### CODE REVIEW

**Comment by krknopp** [comment]
updating to libcurl4, so it doesn't uninstall curl on ubuntu 18.04

**Comment by yohanb** [comment]
Interesting!  Didn't know check mode existed but it makes sense to get the service status.
Thanks for the contribution!

**Comment by egrier** [comment]
Would adding a tag not be more efficient? It's my understanding that using the check mode only applies to running --check or -C when running the entire playbook. If you just wanted to only check the status of the service you could just add a tag of status so only that task will run.

File: tasks/main.yml [hunk 1]
```diff
@@ -62,6 +62,7 @@
   changed_when: false
   failed_when: service_running.rc not in [0,1]
   register: service_running
+  check_mode: no
```
Comment (egrier) [inline tasks/main.yml]:
```suggestion
  check_mode: no
  tags: status
```

**Comment by krknopp** [comment]
Not for my use case.  I want the entire playbook to be able to run in `--check` mode for nightly "checks".  I'm not looking just to check the status of the service.

**Comment by yohanb** [comment]
since this only applies to a specific use case (running a playbook in checkmode), I don't see a problem to leave the `check_mode` attribute.
May I merge the PR @krknopp ?

**Comment by yohanb** [comment]
@krknopp bump

**Comment by krknopp** [comment]
Yes, please do.  Thank you!

**Pull request merged**